rosa_architecture/rosa-sts-roles-and-permissions.adoc

```diff
@@ -0,0 +1,32 @@
+:_content-type: ASSEMBLY
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-sts-ocm-roles-and-permissions
+[id="rosa-sts-understanding-ocm-roles-and-permissions_{context}"]
+= Understanding {cluster-manager} roles and permissions
+
+To create a {product-title} (ROSA) cluster, you must have some `ocm-role` resources and Operator roles and policies created and linked. You use these created roles to provide a point of trust with your AWS account.
+
+== Prerequisites
+
+* You have an AWS account.
+* You installed your preferred command-line interface (CLI) tool.
+
+include::modules/rosa-sts-ocm-roles-and-permissions.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="rosa-sts-understanding-ocm-roles-aws_{context}"]
+== Additional resources
+* link:https://docs.aws.amazon.com/IAM/latest/APIReference/API_Types.html[AWS Identity and Access Management Data Types].
+* link:https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_Types.html[Amazon Elastic Computer Cloud Data Types].
+* link:https://docs.aws.amazon.com/STS/latest/APIReference/API_Types.html[AWS Token Security Service Data Types].
+
+include::modules/rosa-sts-required-roles-and-policies.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="rosa-sts-understanding-ocm-roles-and-permissions-additional-resources_{context}"]
+== Additional resources
```

Comment:
There are two additional resources level 1 headings in this assembly. You could either add all of the references together into one section and place it at the end of the assembly, or alternatively change these H1 headings to dot titles, (i.e. `.Additional resources`), if they need to remain directly under their respective sections.

Reply:
You could add a reference to https://docs.openshift.com/rosa/rosa_architecture/rosa-sts-about-iam-resources.html here, given that the page provides a lot more detail about the account-wide roles and policies that are discussed in the preceding section.